custom/enikshay/messaging/custom_recipients.py

```diff
@@ -40,3 +44,58 @@ def agency_user_case_from_voucher_fulfilled_by_id(handler, reminder):
 
     # get_usercase() just returns None if the user case does not exist; no exceptions raised
     return user.get_usercase()
+
+
+@quickcache(['location.location_id'], timeout=4 * 60 * 60)
+def located_in_mehsana(location):
+    return location.get_ancestors().filter(site_code=LOCATION_SITE_CODE_MEHSANA).count() > 0
```

Comment:
probably not a big deal, but since I think this only refers to a single location (https://enikshay.in/a/enikshay/settings/locations/fa7472fe0c9751e5d14595c1a08bf499/), you could have used the indexed `location_id` property here instead of `site_code`, which I don't think is indexed.

Reply:
it actually won't matter in this case since the heart of the query is going to be the `.get_ancestors()` part, which retrieves the ancestors of the current location. so from there, it's just filtering a handful of locations using site code. but i chose to use site code since i try to allow all constants to be used cross-domain or cross-environment in case this is ever used in a test environment

Reply:
👍